CommonTools/RecoAlgos/interface/TrackSelector.h

```diff
@@ -130,7 +132,7 @@ namespace helper {
 
 
   //----------------------------------------------------------------------
-  class TrackSelectorBase : public edm::EDFilter {
+  class TrackSelectorBase : public edm::stream::EDFilter<> {
   public:
     TrackSelectorBase( const edm::ParameterSet & cfg ) {
       std::string alias( cfg.getParameter<std::string>( "@module_label" ) );
```

Comment:
the template instantiation for the StoreManagerTrait should be updated as well 
see its parameters at 
https://github.com/cms-sw/cmssw/blob/CMSSW_7_5_X/CommonTools/UtilAlgos/interface/StoreManagerTrait.h#L99